Controllers/Test01.php

```diff
@@ -15,7 +15,7 @@ public function __construct(private Engine $engine)
     public function index(Request $request, Response $response): Response
     {
         $items = ['one', 'two', 'three', 'Héllo', 'a@\'t"esté'];
-        $string = $this->engine->renderToString('Test01.latte', ['items' => $items, 'title' => 'From Test01']);
+        $string = $this->engine->renderToString(__DIR__ . '/../templates/Test01.latte', ['items' => $items, 'title' => 'From Test01']);
```

Comment:
🤔 @vaielab has setup base path of templates in https://github.com/vaielab/SlimLattePhpStan/blob/main/public/index.php#L23, you are changing it here because of phpstan latte plugin

Reply:
I see. Maybe we should support configuration of tempalte base path/paths too in phpstan.neon.